csrc/quantization/cutlass_w8a8/scaled_mm_entry.cu

```diff
@@ -102,6 +116,19 @@ bool cutlass_scaled_mm_supports_block_fp8(int64_t cuda_device_capability) {
   return false;
 }
 
+bool cutlass_group_gemm_supported(int64_t cuda_device_capability) {
+  // CUTLASS groped FP8 kernels need at least CUDA 12.0
+  // and at least SM90 (Hopper)
+
+#if defined CUDA_VERSION
+  if (cuda_device_capability >= 90) {
+    return CUDA_VERSION >= 12000;
```

Comment:
I think this needs to be:
```
return CUDA_VERSION >= 12030;
```

see https://github.com/NVIDIA/cutlass/blob/62750a2b75c802660e4894434dc55e839f322277/examples/57_hopper_grouped_gemm/57_hopper_grouped_gemm.cu#L722-L726